client/my-sites/domains/domain-management/components/icann-verification/icann-verification-card.jsx

```diff
@@ -68,6 +68,7 @@ class IcannVerificationCard extends React.Component {
 			<EmailVerificationCard
 				changeEmailHref={ changeEmailHref }
 				contactEmail={ contactDetails.email }
+				headerText={ translate( 'Important: Verify Your Email Address' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Verify your email address' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).